lib/appium.js

```diff
@@ -283,7 +287,7 @@ class AppiumDriver extends BaseDriver {
 // help decide which commands should be proxied to sub-drivers and which
 // should be handled by this, our umbrella driver
 function isAppiumDriverCommand (cmd) {
-  return !isSessionCommand(cmd) || cmd === "deleteSession";
+  return (!isSessionCommand(cmd) || cmd === "deleteSession") && cmd !== 'getStatus';
```

Comment:
I'd rather change it to `!isSessionCommand(cmd) || ['deleteSession', 'getStatus'].indexOf(cmd) >= 0`